nilearn/plotting/tests/test_js_plotting_utils.py

```diff
@@ -295,11 +295,16 @@ def test_temp_file_removing():
     html = js_plotting_utils.HTMLDocument('hello')
     wb_open = webbrowser.open
     webbrowser.open = _open_mock
+    fd, tmpfile = tempfile.mkstemp()
     try:
+        os.close(fd)
+        assert_no_warnings(html.open_in_browser,
+                           file_name=tmpfile, temp_file_lifetime=None)
         html.open_in_browser(temp_file_lifetime=.5)
         assert os.path.isfile(html._temp_file)
         time.sleep(1.5)
         assert not os.path.isfile(html._temp_file)
+        assert_warns(UserWarning, html.open_in_browser, temp_file_lifetime=None)
```

Comment:
Similarly, if in Future a different warning is generated by any of th components, then we will have 2 warnings and this test may fail.

Reply:
same comment as above https://github.com/nilearn/nilearn/pull/2055#discussion_r285024690